### PR TITLE
Add force option

### PR DESCRIPTION
**Pull request by LeonidVas**
Added the ability to skip malformed packages when synchronizing metainformation.

### CODE REVIEW

File: s3repo/model.py [hunk 1]
```diff
@@ -340,6 +340,9 @@ def sync(self, permanent):
                     str(self.s3_settings['region']),
                 ]
 
+                if self.s3_settings.get('force_sync'):
+                    mkrepo_cmd.append('--force')
```
Comment (Totktonada) [inline s3repo/model.py:344]: Is it in effect only on startup? So, we'll decline pushing of a new malformed package?

Comment (LeonidVas) [reply s3repo/model.py:344]: > Is it in effect only on startup? So, we'll decline pushing of a new malformed package?

As I see the use of this option:
1. we  launch the service with the flag and get the lists of malformed packages.
2. we solve problems manually
3. restart the service without the flag

About the effect. The option will be used until the service is restarted.

Comment (Totktonada) [reply s3repo/model.py:344]: Got it. Filed #26 about future enhancements we can do.

File: app.py [hunk 2]
```diff
@@ -57,6 +57,7 @@ def update_cfg_by_env(cfg):
     env_model_settings['base_path'] = os.getenv('S3_BASE_PATH')
     env_model_settings['access_key_id'] = os.getenv('S3_ACCESS_KEY')
     env_model_settings['secret_access_key'] = os.getenv('S3_SECRET_KEY')
+    env_model_settings['force_sync'] = os.getenv('RWS_FORCE_SYNC') == 'True' or False
```
Comment (Totktonada) [inline app.py:60]: Usually any non-empty value of an environment variable is considered as true. It is quite usual to set `FOO=1`. If we'll interpret only `True` as truth, it'll be confusing (especially, if we'll do it case sensitively).

**Comment by Totktonada** [comment]
I'm okay as for the temporary solution that we need today. I would revisit it later and propose something less dangerous. Filed #26, but I guess it is only part of the story about proper failure reporting back to CI.

**Review by Totktonada — APPROVED** [review]

File: app.py (hunk 2 above)
Comment (LeonidVas) [reply app.py:60]: I don't want "False" to mean "True". But I agree that validation can be case insensitive. I will also add an additional note to README.

Comment (LeonidVas) [reply app.py]: Updated.

Comment (Totktonada) [reply app.py]: I don't not want to make `RWS_FORCE_SYNC=1` means `False`.

I experimented around etcd's option `ETCD_ENABLE_V2`. My results:

| environment | result |
| ---- | ---- |
| ETCD_ENABLE_V2=1 | enabled |
| ETCD_ENABLE_V2=true | enabled |
| ETCD_ENABLE_V2=false | disabled |
| ETCD_ENABLE_V2=FALSE | disabled |
| ETCD_ENABLE_V2=0 | disabled |

How to check:

1. Run `ETCD_ENABLE_V2=<...> etcd` in a temporary directory.
2. Run `curl http://127.0.0.1:2379/v2/keys/message`
    - `404 page not found` -> disabled.
    - `{"errorCode":100,"message":"Key not found","cause":"/message","index":<...>}` -> enabled.

My etcd version is 3.4.14.

Comment (LeonidVas) [reply app.py]: Updated.

**Review by Totktonada — APPROVED** [review]
(If you re-asked the review to acquire a feedback about constructing a boolean value from an environment variable value, the proposed behaviour looks okay for me.)

**Pull request merged**